test-cargo-miri/subcrate/Cargo.toml

```diff
@@ -4,6 +4,10 @@ version = "0.1.0"
 authors = ["Miri Team"]
 edition = "2018"
 
+[lib]
+proc-macro = true
```

Comment:
So the lib-crate being a proc-macro crate does not affect the bin-crates at all? I had no idea that's even possible.^^